eessi-2021.12.yml

```
@@ -47,3 +47,7 @@ software:
     toolchains:
       SYSTEM:
         versions: '3.7.3'
+  WRF:
+     toolchains:
+       foss-2020a:
+         versions: ['3.9.1.1']
```

@hmeiland the versionsuffix part is missing, so the WRF easyconfig isn't found:

```
ERROR: One or more files not found: WRF-3.9.1.1-foss-2020a.eb
```

to fix:

```suggestion
         versions:
           '3.9.1.1':
             versionsuffix: -dmpar

---

done